safe/gui/tools/wizard/step_kw40_classify.py

```diff
@@ -92,6 +113,34 @@ def get_next_step(self):
         # Any other case
         return self.parent.step_kw_source
 
+    def inasafe_fields_for_the_layer(self):
```

Comment:
I suggest you to refactor this and put this method in wizard utilities, since there is similar method in the inasafe_field_step.